### ISSUE TITLE
Overfitting problem

### ISSUE THREAD

**Issue by Hcz-up** [akaraspt/deepsleepnet#35]
Why is the accuracy high on the training set but extremely low on the test set and all the data is from sleep-EDF

**Comment by besaman** [comment]
It depends on how "extremely low" is it? is it less than 75% for example?
If it is not, then is not overfitting, but this is the maximum performance you can get with training on the 19 subjects data and test on this particular subject data.
Usually, you get performance of 75-90% accuracy for test data on each fold.
